package.json

```diff
@@ -36,14 +36,13 @@
     "@rollup/plugin-typescript": "11.1.5",
     "@testing-library/react": "14.0.0",
     "@types/chrome": "0.0.248",
-    "@types/jest": "29.5.7",
     "@types/node": "20.8.10",
     "@types/react": "18.2.35",
     "@types/react-dom": "18.2.14",
     "@types/ws": "8.5.8",
     "@typescript-eslint/eslint-plugin": "6.9.1",
     "@typescript-eslint/parser": "6.9.1",
-    "@vitejs/plugin-react": "2.2.0",
+    "@vitejs/plugin-react": "4.1.1",
```

Comment:
You can't update this read more in https://github.com/Jonghakseo/chrome-extension-boilerplate-react-vite/pull/258#issuecomment-1794608629